openreview/conference/templates/programchairWebfield.js

```diff
@@ -270,16 +281,18 @@ var getBlindedNotes = function() {
   return Webfield.getAll('/notes', {
     invitation: BLIND_SUBMISSION_ID,
     details: 'invitation,tags,original,replyCount',
-    sort: 'number:asc'
-  });
+    select: 'id,number,forum,content,details'
+  }).then(function(notes) {
+    return notes;
```

Comment:
I think you can remove this `then` function.

Reply:
Done!